scripts/m18_submission_checker/README.md

```diff
@@ -0,0 +1,22 @@
+# See section 9 of the NIST AIDA 2019 Evaluation Plan V1.3 document for proper archive naming convention, formatting, and archive file types. #
+
+This script is designed to help check if an archive is submit ready for M18 evaluation. 
+Submit ready means that the archive is structured properly for the AIF M18 batch validation process.
+
+This script checks that the ttl files are located in a valid directory structure (i.e. not in the base directory when expected to be in a /NIST/ directory).
+
```

Comment:
Add
This script does not do a complete check against all of the submission archives naming rules.  It checks for the proper submission file name extension and the number of periods in the submission file name.